getting-started/events.md

```diff
@@ -80,73 +80,52 @@ Adds an annotation to the last thread. Keep in mind that sending an annotation c
 `annotation_type` - type of the annotation. This field cannot be empty and is customizable, you can have your own annotation types (i.e. we use "rating" type)
 
 
-<!-- ### Filled form
+### Filled form
 
-A filled form is an event containing some data from a form. Here's what it looks like:
+A filled form is an event containing some data from a form. Let's try on example, we have form:
+
+![Filled Form](../images/filled_form.png "filled form example")
+
+and we want to sent it's data. You can use filled form event to achieve this (in this example we introduce all present field types):
```

Comment:
and we want to sent it's data. You can use filled form event to achieve this (in this example we introduce all present field types): -> and we want to send its data. You can use filled form event to achieve this (in this example we introduce all currently available field types):